README.adoc

```diff
@@ -3,164 +3,159 @@
-listing (from `src/main/java/com/example/schedulingtasks/ScheduledTasks.java`) shows how
-to do so:
+	private static final SimpleDateFormat dateFormat = new SimpleDateFormat("HH:mm:ss");
 
-====
-[source,java]
-----
-include::complete/src/main/java/com/example/schedulingtasks/ScheduledTasks.java[]
+	@Scheduled(fixedRate = 5000)
+	public void reportCurrentTime() {
+		log.info("The time is now {}", dateFormat.format(new Date()));
+	}
+}
 ----
 ====
 
-The `Scheduled` annotation defines when a particular method runs.
+The https://docs.spring.io/spring-framework/reference/integration/scheduling.html#scheduling-annotation-support-scheduled[`Scheduled` annotation^] defines when a particular method runs.
 
-NOTE: This example uses `fixedRate`, which specifies the interval between method
-invocations, measured from the start time of each invocation. There are
-https://docs.spring.io/spring/docs/{spring_version}/spring-framework-reference/html/scheduling.html#scheduling-annotation-support-scheduled[other options],
-such as `fixedDelay`, which specifies the interval between invocations measured from the
-completion of the task. You can also use https://docs.spring.io/spring/docs/{spring_version}/javadoc-api/org/springframework/scheduling/support/CronSequenceGenerator.html[`@Scheduled(cron=". . .")`]
-expressions for more sophisticated task scheduling.
+NOTE: This example uses https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/scheduling/annotation/Scheduled.html#fixedRate()[`fixedRate()`^], which specifies the interval between method
+invocations, measured from the start time of each invocation. Other options are https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/scheduling/annotation/Scheduled.html#cron()[`cron()`^] and https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/scheduling/annotation/Scheduled.html#fixedDelay()[`fixedDelay()`^]. For periodic tasks, exactly one of these three options must be specified, and additionally an optional https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/scheduling/annotation/Scheduled.html#initialDelay()[`initialDelay()`^]. For a one-time task, it is sufficient to just specify an initialDelay()
 
-== Enable Scheduling
+== Running the Application
 
-Although scheduled tasks can be embedded in web applications and WAR files, the simpler
-approach (shown in the next listing) creates a standalone application. To do so,
-package everything in a single, executable JAR file, driven by a good old Java `main()`
-method. The following listing (from
-`src/main/java/com/example/schedulingtasks/SchedulingTasksApplication.java`) shows the
-application class:
+You should now be able to run the application by executing the main method in `SchedulingTasksApplication`. You can run the program from your IDE, or by executing the following Gradle command in the project root directory:
```

Comment:
Change `executing` to `running` (both instances)

Remove the comma after `IDE`